packages/cozy-pouch-link/src/migrations/adapter.js

```diff
@@ -0,0 +1,22 @@
+import PouchDB from 'pouchdb-browser'
+
+const getNewIndexedDBDatabaseName = dbName => {
+  return `${dbName}_indexeddb`
+}
+
+/**
+ * Migrate a PouchDB database to a new adapter.
+ *
+ * @param {object} oldPouch - The pouch to migrate
+ * @param {string} toAdapter - The target adapter name
+ * @returns {object} - The migrated pouch
+ */
+export const migratePouch = async (oldPouch, toAdapter) => {
```

Comment:
I'm not a big fan of this API. First arg is a Pouch Object, second one the name of the Adapter.

Can't we pass within the first arg the name of the adapter? And the name of the old DB? Like that, we only instanciate Pouch in this method not before?

Reply:
Is e265413 better?